core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/htmlviz/HtmlVizOpExec.scala

```diff
@@ -4,15 +4,16 @@ import edu.uci.ics.amber.engine.common.InputExhausted
 import edu.uci.ics.amber.engine.common.virtualidentity.LinkIdentity
 import edu.uci.ics.texera.workflow.common.operators.OperatorExecutor
 import edu.uci.ics.texera.workflow.common.tuple.Tuple
-import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, AttributeType}
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, OperatorSchemaInfo}
 
 import scala.collection.Iterator
 import scala.util.Either
 
 /**
   * HTML Visualization operator to render any given HTML code
   */
-class HtmlVizOpExec(htmlContentAttrName: String) extends OperatorExecutor {
+class HtmlVizOpExec(desc: HtmlVizOpDesc, operatorSchemaInfo: OperatorSchemaInfo)
```

Comment:
why are you passing the descriptor? and not doing that using the config?

Reply:
reverted